samplekotlin/src/main/java/com/example/chattutorial/ChannelActivity4.kt

```diff
@@ -86,11 +87,12 @@ class ChannelActivity4 : AppCompatActivity() {
         ChatClient
             .instance()
             .channel(cid)
-            .subscribeFor(this, TypingStartEvent::class, TypingStopEvent::class
+            .subscribeFor(this, TypingStartEvent::class.java, TypingStopEvent::class.java
```

Comment:
We shouldn't need to change this, we have overloads that accept a `KClass` (as an extension)

Reply:
Nice one! Adding missing import